### ISSUE TITLE
[SF][A11Y][RadioButton]:   Provide interface to customize aria-label

### ISSUE THREAD

**Issue by sfsf-xweb-sh** [SAP/ui5-webcomponents#5516]
**Is your feature request related to a problem? Please describe.**

Hi Colleague,
Please provide an interface for user to customize aria-label.
In our app, **RadioButton** is followed by a label(indicates the count of related option).  We need to add arial-label with pattern: **{0} {1} {2}results found** ( {0} is panel header, {1} is RadioButton text, {2} is the number of results found). For example, in the following screen shot, the aria-label of selected item should be **"Start Date Any Start Date 10 results found"** .
<img width="352" alt="image" src="https://user-images.githubusercontent.com/78520316/179394309-59a6e875-5658-4d73-b9a3-cd505a7fff93.png">

I try to use accessibleNameRef, but the radio button text will appended by default, finally the aria-label is **"Start Date Any Start Date 10 results found Any Start Date"**.

**Describe the solution you'd like**
Provide an interface like accessibleName to customize aria-label

Thanks!
Josephine Su

**Comment by Todor-ads** [comment]
Hello @SAP/ui5-webcomponents-topic-rd  , take a look at this issue.

**Comment by dimovpetar** [comment]
Hello @sfsf-xweb-sh ,

With https://github.com/SAP/ui5-webcomponents/pull/5548 `accessibleName` property has been added to ui5-radio-button.

Best regards,
Petar

**Comment by sfsf-xweb-sh** [comment]
Nicely done! Thanks @dimovpetar
